tests/stockfish/test_models.py

```diff
@@ -270,3 +270,33 @@ def test_get_parameters(self):
         arg1 = s1.get_parameters()
         arg2 = s2.get_parameters()
         assert arg1 != arg2
+
+    def test_get_top_moves(self):
+        stockfish = Stockfish(depth = 15, parameters = {"MultiPV": 3})
+        stockfish.set_fen_position("b5k1/1q3p1p/6p1/8/8/2Q3P1/1B3P1P/6K1 w - - 0 1")
+        top_moves = stockfish.get_top_moves(3)
+        assert ((top_moves[0]["Move"] == "c3g7" and top_moves[1]["Move"] == "c3h8") or
```

Comment:
It seems to me better to do assert for full top_moves

e.g. `assert topmoves == ...`